File: 백준/Bronze/1000. A＋B/A＋B.java
import java.io.*;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int[] nums = Arrays.stream(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();

        bw.write(String.valueOf(nums[0] + nums[1]));
        bw.flush();
        bw.close();
        br.close();
    }
}

File: 백준/Bronze/1000. A＋B/A＋B.kt
import java.io.BufferedWriter
import java.io.OutputStreamWriter

fun main() {
    System.`in`.bufferedReader().use { br ->
        BufferedWriter(OutputStreamWriter(System.out)).use { bw ->
            val nums = br.readLine().split(" ").map { it.toInt() }
            bw.write((nums[0] + nums[1]).toString())
            bw.flush()
        }
    }
}
